src/main/java/com/google/devtools/build/lib/windows/runfiles/WindowsRunfiles.java

```diff
@@ -44,9 +45,12 @@ private static synchronized void ensureRunfilesParsed() throws IOException {
     }
 
     runfiles = new HashMap<>();
-    InputStream fis = new FileInputStream(System.getenv("RUNFILES_MANIFEST_FILE"));
-    InputStreamReader isr = new InputStreamReader(fis, Charset.forName("UTF-8"));
-    try (BufferedReader br = new BufferedReader(isr)) {
+    try (BufferedReader br =
+        new BufferedReader(
+            new InputStreamReader(
+                Files.newInputStream(Paths.get(System.getenv("RUNFILES_MANIFEST_FILE"))),
+                // TODO(laszlocsomor): Fix charset: Bazel writes a Latin-1 manifest file, not UTF-8.
```

Comment:
We probably want to fix this in a follow-up CL, right?

I think the right charset to use here would be StandardCharsets.ISO_8859_1 (because of the whole "let's tunnel file names as bytes through the String datatype" thing), but we should write some tests or an "encoding playground" to ensure that we fully understand what's happening here.

Reply:
Yes, I want to fix this, but I must tread carefully -- it would be a (potentially) breaking change.
I agree we should write some encoding-exercising tests.

Reply:
Oh wait, I got confused, this code is not in the runfiles library (`//tools/java/runfiles`) but in Bazel itself. So I don't think this is a breaking change after all.
In fact, Bazel should use the runfiles library instead of rolling its own runfiles logic. Let me fix that in a follow-up patch.